scripts/publish-to-yalc.mjs

```diff
@@ -0,0 +1,23 @@
+/**
+ * Publish the theatre packages to a local registry with yalc for the build tests.
+ */
+import path from 'path'
+
+const root = path.normalize(path.join(__dirname, '..'))
+
+// Make sure the script runs in the root of the monorepo
+cd(root)
+
+process.env.USING_YALC = 'true'
+
+const packagesToPublish = ['theatre/core', 'theatre/studio']
```

Comment:
Same as the previous comment:
> Any reason we're not also linking `@theatre/dataverse`?

Reply:
They weren't needed for the build_test projects, but eventually they got in: 053e3d09. I'll mark this conversation as 'resolved' too. :relaxed: